### ISSUE TITLE
bug: build failed

### ISSUE THREAD

**Issue by Samarium150** [YunYouJun/valaxy#94]
build failed after `pnpm create`

error log:
```
> npm run build:ssg && npm run rss


> valaxy-blog@0.10.3 build:ssg
> valaxy build --ssg


  🌌 Valaxy  v0.10.3

  🪐 theme   > yun
  📁 C:\Users\Samarium\Documents\projects\valaxy-blog

i use vite-ssg to do ssg build...                                                                                                                                                               13:22:39  

[vite-ssg] Build for client...
vite v2.9.14 building for production...
✓ 160 modules transformed.
[vite]: Rollup failed to resolve import "@waline/client/dist/waline.css?used" from "node_modules/.pnpm/valaxy@0.10.3/node_modules/valaxy/client/composables/comments/waline.ts".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`

 ERROR  [vite-ssg] An internal error occurred.                                                                                                                                                  13:22:41  

Error: [vite]: Rollup failed to resolve import "@waline/client/dist/waline.css?used" from "node_modules/.pnpm/valaxy@0.10.3/node_modules/valaxy/client/composables/comments/waline.ts".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
    at onRollupWarning (C:\Users\Samarium\Documents\projects\valaxy-blog\node_modules\.pnpm\vite@2.9.14_sass@1.54.0\node_modules\vite\dist\node\chunks\dep-c9998dc6.js:41797:19)
    at onwarn (C:\Users\Samarium\Documents\projects\valaxy-blog\node_modules\.pnpm\vite@2.9.14_sass@1.54.0\node_modules\vite\dist\node\chunks\dep-c9998dc6.js:41613:13)
    at Object.onwarn (C:\Users\Samarium\Documents\projects\valaxy-blog\node_modules\.pnpm\rollup@2.77.0\node_modules\rollup\dist\shared\rollup.js:23215:13)
    at ModuleLoader.handleResolveId (C:\Users\Samarium\Documents\projects\valaxy-blog\node_modules\.pnpm\rollup@2.77.0\node_modules\rollup\dist\shared\rollup.js:22465:26)
    at ModuleLoader.resolveDynamicImport (C:\Users\Samarium\Documents\projects\valaxy-blog\node_modules\.pnpm\rollup@2.77.0\node_modules\rollup\dist\shared\rollup.js:22517:120)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async C:\Users\Samarium\Documents\projects\valaxy-blog\node_modules\.pnpm\rollup@2.77.0\node_modules\rollup\dist\shared\rollup.js:22412:32
```

**Comment by TuiMao233** [comment]
You can see #77, which can temporarily solve your problem

**Comment by Samarium150** [comment]
> You can see #77, which temporarily solves your problem, and I think this question is a duplicate

Thanks, solved by adding `"@waline/client": "^2.6.2"` to dependencies. I hope this can be fixed in a later version.

**Comment by YunYouJun** [comment]
After that, I'll separate it.
